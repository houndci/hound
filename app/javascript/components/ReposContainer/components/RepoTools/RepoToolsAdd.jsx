import React from 'react';

const RepoToolsAdd = (props) => (
  <div className="repo-tools-add">
    <a
      className="button repo-tools-add-button"
      href={`https://github.com/apps/${props.appName}/installations/new`}
    >
      Add GitHub repos
    </a>
  </div>
);

export default RepoToolsAdd;
