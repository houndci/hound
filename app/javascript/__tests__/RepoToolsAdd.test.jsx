import RepoToolsAdd from '../components/ReposContainer/components/RepoTools/RepoToolsAdd';

describe('RepoToolsAdd component', () => {
  it('renders the button to add repos', () => {
    const wrapper = shallow(<RepoToolsAdd />);

    expect(wrapper).toMatchSnapshot();
  });
});
