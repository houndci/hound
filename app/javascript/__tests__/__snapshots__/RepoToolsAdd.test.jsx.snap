// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoToolsAdd component renders the button to add repos 1`] = `
<div
  className="repo-tools-add"
>
  <a
    className="button repo-tools-add-button"
    href="https://github.com/apps/undefined/installations/new"
  >
    Add GitHub repos
  </a>
</div>
`;
